installer/PowerToysBootstrapper/bootstrapper/bootstrapper.cpp

```diff
@@ -85,6 +85,10 @@ void show_error_box(const wchar_t* message, const wchar_t* title)
 int bootstrapper(HINSTANCE hInstance)
 {
     winrt::init_apartment();
+    char* programFilesDir = nullptr;
+    size_t size = 0;
+    _dupenv_s(&programFilesDir, &size, "PROGRAMFILES");
+    auto installDir = std::string(programFilesDir) + std::string("\\PowerToys");
```

Comment:
```suggestion
    std::string installDir;
    if (!_dupenv_s(&programFilesDir, &size, "PROGRAMFILES"))
    {
        installDir += programFilesDir;    
        installDir += "\\PowerToys";
    }
```

We should handle the error here. Also, we could rename `installDir` to `defaultInstallDir`